src/mds/FSMap.cc

```diff
@@ -1028,6 +1051,19 @@ void FSMap::insert(const MDSMap::mds_info_t &new_info)
   standby_epochs[new_info.global_id] = epoch;
 }
 
+void FSMap::adjust_standby_fscid(const mds_gid_t standby_gid,
+				 fs_cluster_id_t fscid)
+{
+  assert(standby_daemons.count(standby_gid));
+  standby_daemon_fscid[standby_gid] = fscid;
```

Comment:
`standby_daemon_fscid.at(standby_gid) = fscid;`

is simpler and avoids two lookups.